app/views/handlers/question.py

```diff
@@ -194,7 +226,27 @@ def get_list_summary_context(self):
     def handle_post(self):
         # pylint: disable=no-member
         # wtforms Form parents are not discoverable in the 2.3.3 implementation
-        self.questionnaire_store_updater.update_answers(self.form.data)
+        dynamic_answers = False
+        for item in self.form.data:
+            if list_item_id := item.split("-")[-1]:
```

Comment:
Is there a cleaner way of identifying a dynamic answer? We should already know the answer id/prefix for all dynamic answers - rather than checking the part after the `-` for every item and checking if it's in the list store? We would have the dynamic_answers key in the block at this point